pledgeindia/pledge/utils.py

```diff
@@ -0,0 +1,13 @@
+from .models import *
+
+
+def PledgeData():
+    pledges = Pledge.objects.all()
+    data = []
+    for pledge in pledges:
+        data.append({
+            'title': pledge.title,
+            'description': pledge.description
+        })
```

Comment:
Im not sure why we are returning json here. Wasn't this supposed to be a template view ?

Reply:
If we want to return json, it would be better if we use drf's serializers rather than building the json object like this.

Reply:
This method returns a dictionary with basic Pledge Data. If rendering in template would be required then the homepage view can directly call the method, and if pledges in frontend are loaded via AJAX, then the call to associated view `JsonData` can be used.

We can Use DRF's serializers but I think this and only one other method is required which will return simple `{"success": True}`. So shall we include DRF for 2 small tasks?

Reply:
Okay. Sounds good to me.